pcdsdevices/epics_motor.py

```diff
@@ -221,6 +221,48 @@ def check_limit_switches(self):
         else:
             return "Low [] High []"
 
+    def get_low_lim(self):
+        """Get the low limit."""
+        return self.low_limit_travel.get()
+
+    def set_low_lim(self, value):
+        """
+        Set the low limit.
+
+        Parameters
+        ----------
+        value : float
+            Limit of travel in the negative direction.
+        """
+        if not self.moving:
```

Comment:
Not sure if I need to check if if it is moving or not and if `<=` current position - this is just something the `ophyd` guys are using in `set_lim()`

Reply:
Ah that's interesting- I wonder what kinds of problems can happen from mid-move limit sets...
I'm a bit surprised that ophyd fails silently instead of raising an exception

Reply:
It's a recent addition - not entirely sure why it's there. I'm positive I've set limits when a motor was moving without issue. Perhaps just a race if you set a limit very close to the current position and the readback changes beyond that value? 🤷‍♂️